plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -347,7 +346,7 @@ public Optional<Table> getTable(HiveIdentity identity, String databaseName, Stri
     {
         try {
             return retry()
-                    .stopOn(NoSuchObjectException.class, HiveViewNotSupportedException.class)
```

Comment:
This cleanup makes sense to me.
There are no Trino related calls which could throw `HiveViewNotSupportedException` in the callables wrapped by retry.

Please split the changes from this file in a separate commit.